<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01 Transitional//EN' 'http://www.w3.org/TR/html4/loose.dtd'>
<html>
<head>
  <!-- SchemaSpy rev Unknown -->
  <title>SchemaSpy - View DEVEL31.DB2INST1.XB_EXT_REL_HGNC_RPT_V</title>
  <link rel=stylesheet href='../schemaSpy.css' type='text/css'>
  <meta HTTP-EQUIV='Content-Type' CONTENT='text/html; charset=ISO-8859-1'>
  <SCRIPT LANGUAGE='JavaScript' TYPE='text/javascript' SRC='../jquery.js'></SCRIPT>
  <SCRIPT LANGUAGE='JavaScript' TYPE='text/javascript' SRC='../schemaSpy.js'></SCRIPT>
  <SCRIPT LANGUAGE='JavaScript' TYPE='text/javascript'>
    table='XB_EXT_REL_HGNC_RPT_V';
  </SCRIPT>
</head>
<body>
<table id='headerHolder' cellspacing='0' cellpadding='0'><tr><td>
<div id='header'>
 <ul>
  <li><a href='../index.html' title='All tables and views in the schema'>Tables</a></li>
  <li><a href='../relationships.html' title='Diagram of table relationships'>Relationships</a></li>
  <li><a href='../utilities.html' title='View of tables with neither parents nor children'>Utility&nbsp;Tables</a></li>
  <li><a href='../constraints.html' title='Useful for diagnosing error messages that just give constraint name or number'>Constraints</a></li>
  <li><a href='../anomalies.html' title="Things that might not be quite right">Anomalies</a></li>
  <li><a href='../columns.byTable.html' title="All of the columns in the schema">Columns</a></li>
  <li><a href='http://sourceforge.net/donate/index.php?group_id=137197' title='Please help keep SchemaSpy alive' target='_blank'>Donate</a></li>
 </ul>
</div>
</td></tr></table>
<div class='content' style='clear:both;'>
<table width='100%' border='0' cellpadding='0'>
 <tr>
  <td class='heading' valign='middle'><span class='header'>View <span title='Database'>DEVEL31</span>.<span title='Schema'>DB2INST1</span>.<span title='Table'>XB_EXT_REL_HGNC_RPT_V</span></span></td>
  <td class='heading' align='right' valign='top' title='John Currier - Creator of Cool Tools'><span class='indent'>Generated by</span><br><span class='indent'><span class='signature'><a href='http://schemaspy.sourceforge.net' target='_blank'>SchemaSpy</a></span></span></td>
 </tr>
</table>
<table width='100%' border='0'>
<tr valign='top'><td class='container' align='left' valign='top'>
<form name='options' action=''>
 <label for='implied'><input type=checkbox id='implied' checked>Implied relationships</label>
 <label for='showRelatedCols'><input type=checkbox id='showRelatedCols'>Related columns</label>
 <label for='showConstNames'><input type=checkbox id='showConstNames'>Constraints</label>
 <label for='showComments'><input type=checkbox id='showComments'>Comments</label>
 <label for='showLegend'><input type=checkbox checked id='showLegend'>Legend</label>
</form>
</td><td class='container' rowspan='2' align='right' valign='top'>
 <table class='legend' border='0'>
  <tr>
   <td class='dataTable' valign='bottom'>Legend:</td>
   <td class='container' align='right' valign='top'><a href='http://sourceforge.net' target='_blank'><img src='http://sourceforge.net/sflogo.php?group_id=137197&amp;type=1' alt='SourceForge.net' border='0' height='31' width='88'></a></td>
  </tr>
  <tr><td class='container' colspan='2'>
   <table class='dataTable' border='1'>
    <tbody>
    <tr><td class='primaryKey'>Primary key columns</td></tr>
    <tr><td class='indexedColumn'>Columns with indexes</td></tr>
    <tr class='impliedRelationship'><td class='detail'><span class='impliedRelationship'>Implied relationships</span></td></tr>
    <tr><td class='excludedColumn'>Excluded column relationships</td></tr>
    <tr><td class='legendDetail'>&lt; <em>n</em> &gt; number of related tables</td></tr>
   </table>
  </td></tr>
 </table>
<div style="margin-right: 2pt;">
<script type="text/javascript"><!--
google_ad_client = "pub-9598353634003340";
google_ad_channel ="SchemaSpy-generated";
google_ad_width = 234;
google_ad_height = 60;
google_ad_format = "234x60_as";
google_ad_type = "text";
google_color_border = "9bab96";
google_color_link = "489148";
google_color_text = "000000";
//-->
</script>
<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
</div>
&nbsp;
</td><tr valign='top'><td class='container' align='left' valign='top'>
<a name='columns'></a>
<table id='columns' class='dataTable' border='1' rules='groups'>
<colgroup>
<colgroup>
<colgroup>
<colgroup>
<colgroup>
<colgroup>
<colgroup>
<colgroup>
<colgroup class='comment'>
<thead align='left'>
<tr>
  <th>Column</th>
  <th>Type</th>
  <th>Size</th>
  <th title='Are nulls allowed?'>Nulls</th>
  <th title='Is column automatically updated?'>Auto</th>
  <th title='Default value'>Default</th>
  <th title='Columns in tables that reference this column'><span class='notSortedByColumn'>Children</span></th>
  <th title='Columns in tables that are referenced by this column'><span class='notSortedByColumn'>Parents</span></th>
  <th title='Comments' class='comment'><span class='notSortedByColumn'>Comments</span></th>
</tr>
</thead>
<tbody valign='top'>
<tr class='even'>
 <td class='detail'>XENBASE_GENE_ID</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>19</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='center'></td>
 <td class='detail'></td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='odd'>
 <td class='detail'>SYMBOL</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>255</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='even'>
 <td class='detail'>XENBASE_GENEPAGE_ID</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>23</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='center'></td>
 <td class='detail'></td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='odd'>
 <td class='detail'>XENBASE_TRANSCRIPT_ID</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>4000</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='even'>
 <td class='detail'>ENTREZ_GENE_ID</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>4000</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='odd'>
 <td class='detail'>MODEL_NAME</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>4000</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='even'>
 <td class='detail'>NCBI_GENE_ID</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>255</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='odd'>
 <td class='detail'>DB_ID</td>
 <td class='detail'>integer</td>
 <td class='detail' align='right'>4</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'>
  <table border='0' width='100%' cellspacing='0' cellpadding='0'>
   <tr class='impliedRelationship relative odd' valign='top'>
    <td class='relatedTable detail' title="XB_EXT_REL_HGNC_RPT_V.DB_ID's name implies that it's a child of DB.DB_ID, but it doesn't reference that column."><a href='DB.html'>DB</a><span class='relatedKey'>.DB_ID</span>
    </td>
    <td class='constraint detail'>Implied Constraint<span title='Restrict delete:
 Parent cannot be deleted if children exist'>&nbsp;R</span></td>
   </tr>
  </table>
 </td>
 <td class='comment detail'></td>
</tr>
<tr class='even'>
 <td class='detail'>GENOME_ASSEMBLY</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>255</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='odd'>
 <td class='detail'>ENSEMBL</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>4000</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='even'>
 <td class='detail'>HGNC_ID</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>4000</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='odd'>
 <td class='detail'>ORTHOLOG_HUMAN_SYMBOL</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>4000</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='even'>
 <td class='detail'>ORTHOLOG_HUMAN_ENTREZ</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>4000</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='odd'>
 <td class='detail'>NCBI_TAXON</td>
 <td class='detail'>varchar</td>
 <td class='detail' align='right'>4</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
</table>
<p title='14 columns'>Analyzed at Sun Oct 10 18:11 MDT 2021<p/>
</td></tr></table>
<div class='indent spacer'>
  View Definition:
<div class='viewDefinition preFormatted'>create or replace view db2inst1.xb_ext_rel_hgnc_rpt_v as
WITH genes AS (
	SELECT g.gene_ID, g.symbol, g.organism_id, f.feature_id as genepage_ID, f.full_name as genepage_Name
	FROM genes_MQT g, feature f, feature_relationship fr
	WHERE g.organism_id IN (12,13) AND
			g.gene_id = fr.subject_id AND
			f.feature_id = fr.object_id AND
			fr.type_id = 500 AND
			f.type_id = 1214 AND
			f.organism_id = 11
		--	and g.symbol = 'pax8'
		--	and g.symbol like 'pax%'
		--	and g.symbol like 'tmem184b.L%'
)
,entrez as (
	SELECT g.gene_id, LISTAGG(dbx.accession,'|') as entrez
	FROM genes g, feature_dbxref fdb, dbxref dbx
	WHERE	g.gene_id = fdb.feature_id AND
			fdb.dbxref_id = dbx.dbxref_id AND
			dbx.db_id = 46
	GROUP BY g.gene_id
)
,ensembl as (
	SELECT g.gene_id, LISTAGG(dbx.accession,'|') as ensembl
	FROM dbxref dbx, feature_dbxref fdbx, feature f, feature_relationship fr, genes g
	WHERE
       g.gene_id = fr.object_id AND
       f.feature_id = fr.subject_id AND
       f.feature_id = fdbx.feature_id AND
       fdbx.dbxref_id = dbx.dbxref_id AND
       dbx.db_id = 99 
	GROUP BY g.gene_id
)
,orthohuman as (
	SELECT g.gene_id, LISTAGG(f.name,'|') as ortholog_human_symbol
	, LISTAGG(dbx.accession,'|') as ortholog_human_entrez
	FROM genes g, feature f , feature_relationship fr, feature_dbxref fd , dbxref dbx 
WHERE fr.object_id = g.genepage_ID --483692 -- pax8
AND fr.type_id = 1522 AND fr.subject_id = f.feature_id AND fd.feature_id = f.feature_id AND fd.dbxref_id = dbx.dbxref_id
AND f.organism_id = 1 -- human
AND dbx.db_id = 46 -- entrez gene
	GROUP BY g.gene_id
)
, gene_model AS (
	SELECT g.gene_id, dbx.db_id, db.name dbName, m.name model_name, m.feature_id model_feature_id
	--, LISTAGG(m.name, '|') WITHIN GROUP(ORDER BY m.name DESC) as JGI_models2 
	FROM genes g, feature_relationship fr, feature m, dbxref dbx, db
	WHERE g.gene_id = fr.object_id AND
		m.feature_id = fr.subject_id AND 
		fr.type_id = 67 AND
		dbx.dbxref_id = m.dbxref_id AND
--		dbx.db_id IN (127,131,139,109) -- 103 is XT 4.1, 135 is XT 8.0 neither are really accurate/great
--		dbx.db_id IN (103, 109, 127, 131, 135, 139, 143, 148) -- 109 is XT 7.1, 131 is XL 7.1, 139 is XL9.1, 143 is XT9.1, 148 is XL9.2
		dbx.db_id IN (109, 131, 135, 139, 143, 148) -- 109 is XT 7.1, 131 is XL 7.1, 139 is XL9.1, 143 is XT9.1, 148 is XL9.2 excl. 103 4.1 and 127 6.0
		AND dbx.db_id = db.db_id
--	GROUP BY g.gene_id, dbx.db_id, db.name
), jgi_model
(
gene_id,
db_id,
dbName,
model_name,
--model_geneId,
--model_geneName,
--model_version,
mrnaId,
mrnaName
--, mrnaSeq
--,polypeptideId,
--polypeptideName,
--polypeptideSeq,
--proteinId
) as (
select	
                gm.gene_id,
                gm.db_id,
                gm.dbName,
                gm.model_name,
           --     f1.feature_id as model_geneId,
			--	f1.name as model_geneName,
			--	fp.value as model_version,
				f2.feature_id as mrnaId,
				f2.name as mrnaName
			--	, f2.residues as mrnaSeq
			--	,f3.feature_id as polypeptideId,
			--	f3.name as polypeptideName,
			--	f3.residues as polypeptideSeq,
			--	d2.accession as proteinId
		from	--feature_relationship fr1,
				feature f1,
			--	featureprop fp,
				feature_relationship fr2,
				feature f2,
			--	feature_relationship fr3,
			--	feature f3,
			--	dbxref d2,
				gene_model gm
		where	1=1 AND
				--f1.feature_id = gm.model_feature_id AND
				f1.name = gm.model_name AND
				--f1.name = gm.jgi_id AND   --'Xetro.C00299' AND -- gfra2
		--fr1.object_id=#{geneFeatureId,jdbcType=INTEGER} AND
				--fr1.type_id=134 AND
				--f1.feature_id=fr1.subject_id AND
				--f1.type_id=1214 AND
				--fp.feature_id=f1.feature_id AND
				--fp.type_id=73 AND
				fr2.object_id=f1.feature_id AND
				fr2.type_id=607 AND
				--fr2.r_rank = 1 AND
				f2.feature_id=fr2.subject_id AND
				f2.type_id=738 --AND
				--fr3.object_id=f2.feature_id AND
				--fr3.type_id=607 AND
				--f3.feature_id=fr3.subject_id AND
				--f3.type_id=606 
				-- AND	d2.dbxref_id=f3.dbxref_id

), mrna_transcript AS (
	SELECT g.gene_id, g.db_id, g.dbName, LISTAGG(g.mrnaname, '|') as mRNA_Transcript
	FROM jgi_model g
	WHERE 1=1
	GROUP BY g.gene_id, g.db_id, g.dbName
)
,hgnc as (
	SELECT g.gene_id
	--, LISTAGG(f.name,'|') as ortholog_human_symbol
	, LISTAGG(dbx.accession,'|') as hgnc_id
	FROM genes g, feature f , feature_relationship fr, feature_dbxref fd , dbxref dbx 
WHERE fr.object_id = g.genepage_ID --483692 -- pax8
AND fr.type_id = 1522 AND fr.subject_id = f.feature_id AND fd.feature_id = f.feature_id AND fd.dbxref_id = dbx.dbxref_id
AND f.organism_id = 1 -- human
AND dbx.db_id = 770 -- hgnc
	GROUP BY g.gene_id
)
,JGI AS (
	SELECT g.gene_id, g.db_id, g.dbName, LISTAGG(g.model_name, '|') WITHIN GROUP(ORDER BY g.model_name DESC) as JGI_models 
	
	FROM gene_model g
	GROUP BY g.gene_id, g.db_id, g.dbName
)

SELECT concat('XB-GENE-',g.gene_id) as Xenbase_Gene_ID, g.symbol
--, g.genepage_Name
, concat('XB-GENEPAGE-',g.genepage_ID) as Xenbase_Genepage_id
, mrt.mRNA_Transcript XENBASE_TRANSCRIPT_ID
, e.entrez as ENTREZ_GENE_ID
, jgi.JGI_models model_name
, etl_stage.SPLIT(2,'-', jgi.JGI_models ) ncbi_gene_id
, jgi.db_id
, jgi.dbName genome_assembly
, mbl.ensembl
, r.hgnc_id
, s.ortholog_human_symbol
, s.ortholog_human_entrez
-- exclude from final report output, but needed here to split the data
, CASE WHEN g.organism_id = 13 THEN '8355' WHEN g.organism_id = 12 THEN '8364' END as NCBI_Taxon
FROM genes g
	LEFT JOIN entrez e ON g.gene_id = e.gene_id
	LEFT JOIN orthohuman s ON g.gene_id = s.gene_id
	LEFT JOIN JGI ON g.gene_id = jgi.gene_id
	LEFT JOIN hgnc r ON g.gene_id = r.gene_id
	LEFT JOIN mrna_transcript mrt ON g.gene_id = mrt.gene_id
	LEFT JOIN ensembl mbl ON g.gene_id = mbl.gene_id
WHERE jgi.db_id = mrt.db_id</div>
</div>
<div class='spacer'>&nbsp;</div>
<div class='indent'>
  Possibly Referenced Tables/Views:
  <div class='viewReferences'>
  <a href='DB.html'>DB</a>&nbsp;<a href='DBXREF.html'>DBXREF</a>&nbsp;<a href='FEATURE.html'>FEATURE</a>&nbsp;<a href='FEATURE_DBXREF.html'>FEATURE_DBXREF</a>&nbsp;<a href='FEATURE_RELATIONSHIP.html'>FEATURE_RELATIONSHIP</a>&nbsp;<a href='FEATUREPROP.html'>FEATUREPROP</a>&nbsp;<a href='GENES_MQT.html'>GENES_MQT</a>&nbsp;<a href='XB_EXT_REL_HGNC_RPT_V.html'>XB_EXT_REL_HGNC_RPT_V</a>&nbsp;  </div>
</div><p/>
<table width='100%' border='0'><tr><td class='container'>
<br><form action='get'><b>Close relationships:</b></form><map id="oneDegreeRelationshipsDiagram" name="oneDegreeRelationshipsDiagram">
<area shape="rect" id="node1" href="XB_EXT_REL_HGNC_RPT_V.html" title="XB_EXT_REL_HGNC_RPT_V" alt="" coords="5,5,349,405">
</map>
  <a name='diagram'><img id='oneDegreeImg' src='../diagrams/XB_EXT_REL_HGNC_RPT_V.1degree.png' usemap='#oneDegreeRelationshipsDiagram' class='diagram' border='0' alt='' align='left'></a>
<map id="impliedTwoDegreesRelationshipsDiagram" name="impliedTwoDegreesRelationshipsDiagram">
<area shape="rect" id="node1" href="ATTRIB_ARTICLE_RECON_V.html" title="ATTRIB_ARTICLE_RECON_V" alt="" coords="943,780,1172,863">
<area shape="rect" id="node2" href="DB.html" title="DB" alt="" coords="219,1007,463,1377">
<area shape="rect" id="node3" href="DBXREF.html" title="DBXREF" alt="" coords="580,1111,777,1193">
<area shape="rect" id="node9" href="ORGANIZATION.html" title="ORGANIZATION" alt="" coords="5,1293,175,1376">
<area shape="rect" id="node4" href="ATTRIB_ARTICLE_REF.html" title="ATTRIB_ARTICLE_REF" alt="" coords="963,880,1151,963">
<area shape="rect" id="node5" href="ATTRIB_ARTICLE_REF_MQT.html" title="ATTRIB_ARTICLE_REF_MQT" alt="" coords="943,980,1171,1063">
<area shape="rect" id="node6" href="ATTRIB_ARTICLE_REF_STG.html" title="ATTRIB_ARTICLE_REF_STG" alt="" coords="944,1080,1171,1163">
<area shape="rect" id="node7" href="ATTRIB_DO_ARTICLE_LIST_MQT.html" title="ATTRIB_DO_ARTICLE_LIST_MQT" alt="" coords="928,1180,1187,1263">
<area shape="rect" id="node8" href="AUDIT_DB.html" title="AUDIT_DB" alt="" coords="600,5,757,88">
<area shape="rect" id="node10" href="AUDIT_DBXREF.html" title="AUDIT_DBXREF" alt="" coords="965,1280,1150,1363">
<area shape="rect" id="node11" href="AUDIT_SUPPLIER_DB.html" title="AUDIT_SUPPLIER_DB" alt="" coords="967,123,1148,205">
<area shape="rect" id="node12" href="SUPPLIER_DB.html" title="SUPPLIER_DB" alt="" coords="612,141,745,224">
<area shape="rect" id="node13" href="DB_INCL_DEL_V.html" title="DB_INCL_DEL_V" alt="" coords="606,1427,751,1509">
<area shape="rect" id="node14" href="DB_SINCE_LAST_SRCHINDX_GEN_V.html" title="DB_SINCE_LAST_SRCHINDX_GEN_V" alt="" coords="535,1527,822,1609">
<area shape="rect" id="node15" href="DBXREF_EXC.html" title="DBXREF_EXC" alt="" coords="993,1380,1121,1463">
<area shape="rect" id="node16" href="DBXREF_INCL_DEL_V.html" title="DBXREF_INCL_DEL_V" alt="" coords="965,1480,1149,1563">
<area shape="rect" id="node17" href="DBXREF_SINCE_LAST_SRCHINDX_GEN_V.html" title="DBXREF_SINCE_LAST_SRCHINDX_GEN_V" alt="" coords="895,1580,1220,1663">
<area shape="rect" id="node18" href="DISEASE_ARTICLE_LIST_V.html" title="DISEASE_ARTICLE_LIST_V" alt="" coords="949,229,1166,312">
<area shape="rect" id="node19" href="DISEASE_ARTICLE_STATS_V.html" title="DISEASE_ARTICLE_STATS_V" alt="" coords="940,343,1175,425">
<area shape="rect" id="node20" href="GENE_HP_MQT.html" title="GENE_HP_MQT" alt="" coords="975,469,1140,552">
<area shape="rect" id="node21" href="GENE_MP_MQT.html" title="GENE_MP_MQT" alt="" coords="975,624,1140,707">
<area shape="rect" id="node22" href="GENOME_DRAFTS.html" title="GENOME_DRAFTS" alt="" coords="598,385,759,468">
<area shape="rect" id="node23" href="XAO_GP_STAGE.html" title="XAO_GP_STAGE" alt="" coords="605,485,752,568">
<area shape="rect" id="node24" href="XB_EXT_REL_HGNC_RPT_V.html" title="XB_EXT_REL_HGNC_RPT_V" alt="" coords="507,585,851,985">
</map>

  <a name='diagram'><img id='impliedTwoDegreesImg' src='../diagrams/XB_EXT_REL_HGNC_RPT_V.implied2degrees.png' usemap='#impliedTwoDegreesRelationshipsDiagram' class='diagram' border='0' alt='' align='left'></a>
</td></tr></table>
</div>
</body>
</html>
